tools/splat.yaml

```diff
@@ -1151,8 +1151,10 @@ segments:
     start: 0x8EB360
     vram: 0x80240000
     files:
-      - [0x8EB360, "c", "world/area_kmr/kmr_12"]
-      - [0x8EB390, "bin", "world/area_kmr/kmr_12_data"]
+      - [0x8EB360, "c", "world/area_kmr/kmr_12/header"]
+      - [0x8EB390, ".data", "world/area_kmr/kmr_12/header"]
```

Comment:
is there actually padding between these two data segments? What's going on? Potentially a comment here for future would be good.

Reply:
Yes, 1 byte of padding.